pkgs/cybozu/assam/registry.yaml

```diff
@@ -0,0 +1,27 @@
+packages:
+  - type: github_release
+    repo_owner: cybozu
+    repo_name: assam
+    asset: assam_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+    format: tar.gz
+    description: Get a credential by AssumeRoleWithSAML for AWS CLI and SDK
+    replacements:
+      amd64: x86_64
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+    overrides:
+      - goos: windows
+        format: zip
+    supported_envs:
+      - darwin
+      - amd64
+      - linux
```

Comment:
```suggestion
```

linux/arm64 isn't supported.
https://github.com/cybozu/assam/releases/tag/v1.2.6

Reply:
```console
$ ❯ aqua gr cybozu/assam
packages:
  - type: github_release
    repo_owner: cybozu
    repo_name: assam
    asset: assam_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
    format: tar.gz
    description: Get a credential by AssumeRoleWithSAML for AWS CLI and SDK
    replacements:
      amd64: x86_64
      darwin: Darwin
      linux: Linux
      windows: Windows
    overrides:
      - goos: windows
        format: zip
    supported_envs:
      - darwin
      - amd64
    checksum:
      type: github_release
      asset: checksums.txt
      file_format: regexp
      algorithm: sha256
      pattern:
        checksum: "^(\\b[A-Fa-f0-9]{64}\\b)"
        file: "^\\b[A-Fa-f0-9]{64}\\b\\s+(\\S+)$"
```

It seems you add `linux` manually, but you shouldn't add `linux` because linux/arm64 isn't supported.
`linux/amd64` is supported by `amd64`, so you don't have to add `linux`.

Reply:
Thank you for Thank you for being so helpful and reviewing!

> So it seems you add linux manually, but you shouldn't add linux because linux/arm64 isn't supported.
> linux/amd64 is supported by amd64, so you don't have to add linux.

OK, I understand 👌